uaclient/contract.py

```diff
@@ -88,7 +84,6 @@ def request_account_contracts(self, macaroon_token, account_id):
         headers = self.headers()
         headers.update({'Authorization': 'Macaroon %s' % macaroon_token})
         account_contracts, _headers = self.request_url(url, headers=headers)
-        self.cfg.write_cache('account-contracts', account_contracts)
```

Comment:
I'd like to leave this cache file in place for a couple of reasons:
 - I find value in having a cached reference for all ua-contracts responses to aid in debug/triage
 - we are only a few days from starting mulit-contract support and we'll need to be referencing this response file during ua attach to select/reselect the proper contract during attach.

I suppose these are both circumstantial requests, and could be resolved other ways, but minimally I'd like to wait when dropping cache files until we have a better way of preserving ua-contracts response than log files. We also may have more fodder to drop this once multi-contract feature is started.

Reply:
OK, I'm not sure I agree with the reasoning here, but in the interests of landing everything else I'll un-drop this and we can pick it up separately.